backend/inspirehep/records/marshmallow/journals/ui.py

```diff
@@ -0,0 +1,32 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright (C) 2022 CERN.
+#
+# inspirehep is free software; you can redistribute it and/or modify it under
+# the terms of the MIT License; see LICENSE file for more details.
+from marshmallow import fields
+
+from inspirehep.records.marshmallow.fields.nested_without_empty_objects import (
+    NestedField,
+)
+from inspirehep.records.marshmallow.journals.base import JournalsPublicSchema
+
+from inspirehep.records.marshmallow.utils import get_first_value_for_schema
+from inspirehep.search.api import JournalsSearch
+
+
+class JournalsBaseSchema(JournalsPublicSchema):
+    pass
+
+
+class JournalsDetailSchema(JournalsBaseSchema):
+    public_notes = fields.Method("", dump_only=True) 
+    short_title = fields.Method("", dump_only=True) 
+    publisher = fields.Method("", dump_only=True)
+    journal_title = fields.Method("", dump_only=True)
+    urls = fields.Method("", dump_only=True)
```

Comment:
again, using `fields.method` is wrong here. Also, you are copy-pasting almost everything from `JournalsPublicListSchema`, could you use inheritance instead?